Support/Testing/CircleCI/Dockerfile

```diff
@@ -40,6 +40,35 @@ MAINTAINER Stephane Sezer <sas@fb.com>
 RUN apt-get update
 RUN apt-get install -y software-properties-common python-software-properties wget
 
+# Incase lldb needs built
+RUN apt-get install -y libz-dev
+RUN apt-get install -y swig
+RUN apt-get install -y ncurses-dev
```

Comment:
You can put these on just one line, it'll be faster to install.

`RUN apt-get install -y libz-dev ncurses-dev swig`